pos_payment_restriction/static/src/js/models.js

```diff
@@ -0,0 +1,15 @@
+/*
+    Copyright 2019 ACSONE SA/NV
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+*/
+
+odoo.define('pos_payment_restriction.models', function (require) {
+    "use strict";
+
+    var models = require('point_of_sale.models');
+
+    var _paymentlineproto = models.Paymentline.prototype;
+    models.Paymentline = models.Paymentline.extend({
+    });
```

Comment:
I guess this is WIP, isn't it? :sweat_smile:

Reply:
Done